tests/test_gbdesAstrometricFit.py

```diff
@@ -1058,6 +1061,114 @@ def test_inputCameraModel(self):
                     self.assertAlmostEqual(np.std(dDec), 0)
 
 
+class TestApparentMotion(lsst.utils.tests.TestCase):
+    """This class simulates an array of objects with random proper motions and
+    parallaxes and compares the results of
+    `lsst.drp.tasks.gbdesAstrometricFit.calculate_apparent_motion` against the
+    results calculated by astropy."""
+
+    @classmethod
+    def setUpClass(cls):
+        np.random.seed(12345)
+
+        cls.RAs = np.random.rand(10) + 150.0
+        cls.Decs = np.random.rand(10) + 2.5
+
+        cls.pmRAs = np.random.random(10) * 10
+        cls.pmDecs = np.random.random(10) * 10
+
+        cls.parallaxes = abs(np.random.random(10) * 5)
+
+        cls.mjds = np.random.rand(10) * 20 + 57000
+
+    def testProperMotion(self):
+        """Calculate the change in position due to proper motion for a given
+        time change, and compare the results against astropy.
+        """
+
+        parallaxes = np.zeros(len(self.RAs))
+        refEpoch = 57100
+
+        data = pd.DataFrame(
+            {
+                "ra": self.RAs,
+                "dec": self.Decs,
+                "pmRA": self.pmRAs,
+                "pmDec": self.pmDecs,
+                "parallax": parallaxes,
+                "MJD": self.mjds,
+            }
+        )
+        ra_correction, dec_correction = calculate_apparent_motion(data, refEpoch)
+
+        pmRA_cosDec = self.pmRAs * np.cos((self.Decs * u.degree).to(u.radian))
+        coords = SkyCoord(
+            ra=self.RAs * u.degree,
+            dec=self.Decs * u.degree,
+            pm_ra_cosdec=pmRA_cosDec * u.mas / u.yr,
```

Comment:
Why do you multiply by `u.mas` here, it's already a Quantity above, no? Why not do this?

```suggestion
            pm_ra_cosdec=self.pm_ra * np.cos((self.decs * u.degree) / u.yr
```

Reply:
I guess I don't need to convert to radians at line 1104, but after you do `np.cos((self.decs * u.degree)`, you have a dimensionless quantity, so it still needs to have the `u.mas`.